src/sentry/migrations/0359_duplicate_perf_issue_events_issue_platform.py

```diff
@@ -0,0 +1,222 @@
+# Generated by Django 2.2.28 on 2023-02-14 19:42
+from typing import Any, Mapping, Sequence, Tuple
+
+from django.db import migrations
+
+from sentry.event_manager import GroupInfo
+from sentry.eventstore.models import Event
+from sentry.issues.ingest import process_occurrence_data
+from sentry.issues.issue_occurrence import IssueOccurrence, IssueOccurrenceData
+from sentry.models import Group
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.snuba.dataset import Dataset, EntityKey
+
+
+def backfill_eventstream(apps, schema_editor):
+    """
+    Backfills Performance-issue events from the transaction table to the IssuePlatform dataset(search_issues).
+    1. Read all transactions that back a performance-issue(s) with the following criteria:
+        a. non-null group_ids column
+        b.
+    2. Prepare and format each transaction to be submitted through the eventstream.
+    3. Send it through the eventstream.
+    """
+
+    # TODO: need to make sure event_type is correct in eventstream.send()
+    from sentry.issues.ingest import send_issue_occurrence_to_eventstream
+
+    # retrieve rows
+    rows = _query_performance_issue_events()
+
+    for row in rows:
+        try:
+            # don't need to store in node_store since it should be there already, maybe check and write if doesn't exists
```

Comment:
No need to check. All perf issues are associated with an existing transaction event